### ISSUE TITLE
Change button ring color according to the background color of the button

### ISSUE THREAD

**Issue by Lauritz-Tieste** [Bootstrap-Academy/Bootstrap-Academy#92]
Currently, the color of the hover effect of buttons does not match the color of the border. The border colors of the buttons are defined with TailwindCSS variables. Here it is required that the color of the hover effect (ring) matches the border color.

**Expected behavior:** When a user moves the mouse pointer over a button, the color of the ring or border should match the border color of the button defined by TailwindCSS variables.

This is what it looks like at the moment:
![image](https://github.com/Bootstrap-Academy/Bootstrap-Academy/assets/84938977/b6d25b22-c693-43e7-8b4d-352fa4a4c93f)

**Comment by Piitschy** [comment]
If the border and the background have different colors, which one should be matched?

**Comment by Lauritz-Tieste** [comment]
Coachings should have blue border, webinars yellow ones ...

**Comment by realcr33p3r** [comment]
Hello everyone,
I’m new here (apologies if I seem a bit clueless)! I created something about a year ago, and I want to know, what you think:
![image](https://github.com/user-attachments/assets/50a0b8d8-6615-4cce-835f-222739b8029e)

**Comment by realcr33p3r** [comment]
You mean something like that?:
<script src="https://cdn.tailwindcss.com"></script>
<button class="btn px-4 py-2 rounded-lg border-2 border-blue-500 bg-transparent text-black hover:ring-4 hover:ring-blue-500 uppercase transition">Button 1</button><br><br>

**Comment by Lauritz-Tieste** [comment]
The Bootstrap Academy has a folder with centrally defined components with the name `components`. These are reused by all pages. There is the component `components/Btn.vue`. The changes must be integrated there, no new component needs to be developed.

**Comment by realcr33p3r** [comment]
Ahhh ok, I understand. tysm

**Comment by LennartKDeveloper** [comment]
Is this one still unsolved? If yes i would like to try it :)

**Comment by CK-IT-Solutions** [comment]
I would like to solve this as part of the hackathon.

**Comment by Lauritz-Tieste** [comment]
I think @realcr33p3r is also interested in this and came first

**Comment by realcr33p3r** [comment]
Yes, but has it started yet? I thought it only starts on October 27th, am I wrong? Also, I'm not sure if I can issue an invoice if I solve it.

**Comment by TheMorpheus407** [comment]
Hackathon starts Monday, but we can assign tickets already 😊
About the invoice: you should have a registered business for that, if not, we can only assign Morphcoin if you solve the issue. However, if you just like to solve it, you don't need a business. That's only necessary for payment.

**Comment by realcr33p3r** [comment]
I want to try to solve it. But I am not sure that I can so it.

**Comment by realcr33p3r** [comment]
@Lauritz-Tieste  The picture that was shown at the beginning, where are these different colored buttons?

**Comment by Lauritz-Tieste** [comment]
On the rootSkillTree > RootSkill > SubSkill page

https://test.bootstrap.academy/skill-tree/start/medienkompetenz

**Comment by realcr33p3r** [comment]
@Lauritz-Tieste tysm

**Comment by CK-IT-Solutions** [comment]
@realcr33p3r If you need any help feel free to ask.

**Comment by realcr33p3r** [comment]
I want @CK-IT-Solutions to do it when he wants because he can earn money, etc. I'll give it a try, but he can also do it for the hackathon! Good luck!

**Comment by TheMorpheus407** [comment]
You can do it together, if you want 😊 or Collab on discord etc

**Comment by CK-IT-Solutions** [comment]
@realcr33p3r I am also working on another ticket so you can try it and if you have any question you can reach out to me. Then I can try to help you.

**Comment by PeachpitRipley** [comment]
Hello, I would like to take over #92. In the project overview, I see no activity since October 2024 and no pull request. Would it be possible to assign this task to me? Thank you

**Comment by PeachpitRipley** [comment]
I started working on the ticket. Unfortunately, I cannot reproduce the error or test the new code in dev, because my login is not accepted.
„Alle Accounts von the-morpheus.cc wurden in die Bootstrap Academy migriert, du kannst dich also weiterhin mit deinem alten Coding Challenges-Account anmelden.“ does not work.

**Comment by Lauritz-Tieste** [comment]
From the README.md of the frontend repo:

Note on Account Creation
You need to create separate accounts for test instances (localhost, https://test.bootstrap.academy, PullRequest-preview pages) and live instances (https://bootstrap.academy). These are two separate database systems.
